LocalFeed/src/netfx_NETCoreSDK/IlcComponents.xslt

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+	xmlns:wix="http://schemas.microsoft.com/wix/2006/wi"
+  xmlns:netfx="http://schemas.microsoft.com/wix/NetFxExtension">
+
+  <xsl:output method="xml" encoding="UTF-8" indent="yes"/>
+  
+  <xsl:param name="blacklist" select="document('blacklist.xml')/BlackList/Assembly"/>
```

Comment:
This term should be avoided. Replace with ExcludeList or BlockList or something.